crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -536,28 +536,40 @@ static const uint8_t kAESGCMCiphertext_256[64] = {
 };
 
 struct AEADTestVector {
+  const char* name;
   const EVP_AEAD *cipher;
   const uint8_t *key;
   const int key_length;
   const uint8_t *expected_ciphertext;
   const int cipher_text_length;
   const int expect_approved;
+  const bool test_repeat_nonce;
 } nAEADTestVectors[] = {
     // Internal IV usage of AES-GCM is approved.
-    { EVP_aead_aes_128_gcm_randnonce(), kAESKey, 16, nullptr, 0, AWSLC_APPROVED },
-    { EVP_aead_aes_256_gcm_randnonce(), kAESKey_256, 32, nullptr, 0, AWSLC_APPROVED },
+    { "AES-GCM 128-bit key internal iv test", EVP_aead_aes_128_gcm_randnonce(),
+        kAESKey, 16, nullptr, 0, AWSLC_APPROVED, false },
+    { "AES-GCM 256-bit key internal iv test", EVP_aead_aes_256_gcm_randnonce(),
```

Comment:
Should we test that 192 is not approved?

Reply:
`EVP_aead_aes_192_gcm_randnonce` doesn't exist, which was one of the reasons we decided not to approve it at the time.